software/tabix/bgzip/main.nf

```diff
@@ -4,31 +4,31 @@ include { initOptions; saveFiles; getSoftwareName } from './functions'
 params.options = [:]
 def options    = initOptions(params.options)
 
-process BCFTOOLS_BGZIP {
+process TABIX_BGZIP {
     tag "$meta.id"
     publishDir "${params.outdir}",
         mode: params.publish_dir_mode,
         saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:meta.id) }
 
-    conda (params.enable_conda ? "bioconda::bcftools=1.11=h7c999a4_0" : null)
+    conda (params.enable_conda ? "bioconda::tabix=0.2.6" : null)
     if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
-        container "https://depot.galaxyproject.org/singularity/bcftools:1.11--h7c999a4_0"
+        container "https://depot.galaxyproject.org/singularity/tabix:0.2.6--ha92aebf_0"
     } else {
-        container "quay.io/biocontainers/bcftools:1.11--h7c999a4_0"
+        container "quay.io/biocontainers/tabix:0.2.6--ha92aebf_0"
     }
 
     input:
-    tuple val(meta), path(vcf)
+    tuple val(meta), path(file)
 
     output:
-    tuple val(meta), path("*.gz"), emit: vcf
+    tuple val(meta), path("*.gz"), emit: file
```

Comment:
I always get paranoid about using variables that are native language variables.

Reply:
If it works...it works 🤷🏽

Reply:
good point, I'll change that for `gz`